internal/enum/enum.go

```diff
@@ -187,12 +225,17 @@ func Main(config string) error {
 		return err
 	}
 
-	out, err := os.Create(input.Path)
+	var buf bytes.Buffer
+	if err := tmpl.Execute(&buf, input); err != nil {
+		return err
+	}
+
+	formatted, err := format.Source(buf.Bytes())
 	if err != nil {
 		return err
 	}
-	defer out.Close()
-	return tmpl.ExecuteTemplate(out, "enum.go.tmpl", input)
+
+	return os.WriteFile(input.Path, formatted, osx.PermAR|osx.PermGW)
```

Comment:
Pretty sure you meant "user" write, not group (`PermGW` -> `PermUW`). However, I'd also suggest doing what most simple unix tools do and use `PermAW` (effectively 0666 permissions), which lets the process umask strip away the unwanted bits.